resources/user-agents/browsers/lovense-browser/lovense-browser.json

```diff
@@ -0,0 +1,53 @@
+{
+  "division": "Lovense Browser",
+  "sortIndex": 946,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Lovense Browser for Windows",
+      "browser": "lovense-browser",
+      "platform": "Windows",
+      "engine": "Blink",
+      "device": "Windows Desktop",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Lovense Browser",
+        "Alpha": "false",
+        "Beta": "false"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Chrome/* Safari/* Lovense/*",
+          "device": "Windows Desktop",
+          "platforms": [
+            "Win10_x64_B", "Win10_64_B", "Win10_32_B",
```

Comment:
I don't think we've seen any native 64 bit versions of this browser, but I can re-search the logs.

Reply:
This is all we've seen in 2017 that fits that format (windows UAs):

```
Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.87 Safari/537.36 Lovense/1.0.2.8
Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.87 Safari/537.36 Lovense/1.0.3.0
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.87 Safari/537.36 Lovense/10.0.3.5
```

We haven't seen any since November, which I assume is related to them removing the `Lovense` identifier (that I noticed in my testing).

Reply:
Test case `issue-1715-D` is the version I installed today.